app/vminsert/common/streamaggr.go

```diff
@@ -59,6 +59,7 @@ func CheckStreamAggrConfig() error {
 //
 // MustStopStreamAggr must be called when stream aggr is no longer needed.
 func InitStreamAggr() {
+	saCfgReloaderStopCh = make(chan struct{})
```

Comment:
I need to do this because there are sequent test cases under [unittest_test.go](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/ffb6d695af01ac8292b67f090f60d2305a47a5da/app/vmalert/unitest_test.go#L14-L30), and the unit test will use [vminsert.Stop()](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/add-rule-test/app/vminsert/main.go#L103) to clean the environment after each test. So the saCfgReloaderStopCh will be closed again if it's defined globally. 
But I think that won't be a problem for users, cause vmalert will finish one unit test and exit.